src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { keysToRemove, extraData, type } = action;
+
+    switch (true) {
+      case type === 'addProperties':
+        Object.assign(state, extraData);
+        break;
+
+      case type === 'removeProperties':
+        for (const i of keysToRemove) {
```

Comment:
```suggestion
        for (const key of keysToRemove) {
```
use meaningful names